app/services/register_user.rb

```diff
@@ -0,0 +1,11 @@
+class RegisterUser
+  attr_reader :github_id
+
+  def initialize(params)
+    @github_id = params[:github_id]
+  end
+
+  def perform
+    @_user = User.create(github_id: github_id)
```

Comment:
I guess it couldn't hurt to memoize. It seems like it would be Bad to call `SomeServiceClass#perform` more than once, so I guess memoizing protects us from that.